src/main/java/software/aws/neptune/jdbc/utilities/ConnectionProperties.java

```diff
@@ -40,24 +41,26 @@
  * Class that contains connection properties.
  */
 public abstract class ConnectionProperties extends Properties {
-    public static final String SCAN_TYPE_KEY = "ScanType";
-    public static final String APPLICATION_NAME_KEY = "ApplicationName";
-    public static final String AUTH_SCHEME_KEY = "AuthScheme";
-    public static final String CONNECTION_TIMEOUT_MILLIS_KEY = "ConnectionTimeout";
-    public static final String CONNECTION_RETRY_COUNT_KEY = "ConnectionRetryCount";
-    public static final String LOG_LEVEL_KEY = "LogLevel";
+    public static final String SCAN_TYPE_KEY = "scanType";
```

Comment:
We should ensure documentation is also updated.

Reply:
Created a new ticket for it. But I am updating this in the next push.